### PR TITLE
Remove Travis and use Github Actions

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

**Comment by erikzhang** [comment]
Wait for #154

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        dotnet-version: ${{ env.DOTNET_VERSION }}
+    - name: Check format
+      run: |
+        dotnet tool install --tool-path ./ dotnet-format
+        ./dotnet-format --check --dry-run -v diagnostic
+    - name: Test
+      run: |
+        find tests -name *.csproj | xargs -I % dotnet add % package coverlet.msbuild
+        dotnet test tests/Neo.Compiler.MSIL.UnitTests /p:CollectCoverage=true /p:CoverletOutput=${GITHUB_WORKSPACE}/coverage/ /p:Exclude="[Neo.SmartContract.*]*"
+        dotnet test tests/Neo.SmartContract.Framework.UnitTests /p:CollectCoverage=true /p:CoverletOutput=${GITHUB_WORKSPACE}/coverage/lcov /p:MergeWith=${GITHUB_WORKSPACE}/coverage/coverage.json /p:Exclude="[Neo.SmartContract.*]*" /p:CoverletOutputFormat=lcov
```
Comment (vncoelho) [inline .github/workflows/main.yml]: Maybe divide these into Test1 and Test2 might be good.

Comment (erikzhang) [reply .github/workflows/main.yml]: What do you mean?

Comment (vncoelho) [reply .github/workflows/main.yml]: Something like this:

```yml
    - name: TestCompiler
      run: |
        find tests -name *.csproj | xargs -I % dotnet add % package coverlet.msbuild
        dotnet test tests/Neo.Compiler.MSIL.UnitTests /p:CollectCoverage=true /p:CoverletOutput=${GITHUB_WORKSPACE}/coverage/ /p:Exclude="[Neo.SmartContract.*]*"
    - name: TestSC
      run: |
        find tests -name *.csproj | xargs -I % dotnet add % package coverlet.msbuild
        dotnet test tests/Neo.SmartContract.Framework.UnitTests /p:CollectCoverage=true /p:CoverletOutput=${GITHUB_WORKSPACE}/coverage/lcov /p:MergeWith=${GITHUB_WORKSPACE}/coverage/coverage.json /p:Exclude="[Neo.SmartContract.*]*" /p:CoverletOutputFormat=lcov
```

Comment (erikzhang) [reply .github/workflows/main.yml]: What's the difference?

**Comment by devhawk** [comment]
@erikzhang #154 is still in draft while I work on the corresponding changes in neo-debugger. Go ahead with this PR if you're ready. I don't think this PR will affect my PR since mine is going into master 2.x

File: .github/workflows/main.yml (hunk 1 above)
Comment (vncoelho) [reply .github/workflows/main.yml]: Minor but has some difference when you access the workflow status.
We could easily see which of them failed without looking at the logs.

Comment (erikzhang) [reply .github/workflows/main.yml]: Done.

**Pull request merged**